stock-tracker/src/features/latestNews/redux/actions.ts

```diff
@@ -1,26 +1,33 @@
 import { SET_NEWS, SET_LOADING_NEWS, SET_ERROR_NEWS } from "./actionTypes";
 import { Action, ActionPayload } from "../../../types";
 
-export interface NewsType {
+export interface NewsInterface {
   url: string;
   headline: string;
   source: string;
   datetime: string;
 }
 
-type SetLatestNews = ActionPayload<typeof SET_NEWS, NewsType[]>;
-type SetLoading = Action<typeof SET_LOADING_NEWS>;
-type SetError = Action<typeof SET_ERROR_NEWS>;
+export type SetLatestNews = ActionPayload<typeof SET_NEWS, NewsInterface[]>;
+export type SetLatestNewsLoading = Action<typeof SET_LOADING_NEWS>;
+export type SetLatestNewsError = Action<typeof SET_ERROR_NEWS>;
```

Comment:
Rather than bunching these all together towards the top of the file, it might be nice to place them above each action.

E.g.
`export type SetLatestNews = ActionPayload<typeof SET_NEWS, NewsInterface[]>;`

`export const setLatestNewsAction = (`
`  latestNews: NewsInterface[]`
`): SetLatestNews => ({`

`export type SetLatestNewsLoading = Action<typeof SET_LOADING_NEWS>;`